prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-email-view/notification-email-view.component.html

```diff
@@ -0,0 +1,20 @@
+<app-page-subheader *ngIf="emailTemplate"
+                    class="mb-4"
+                    [divider]="false">
+  <ng-container appPageSubheaderTitle>
+    {{ EmailTemplateTypeEnum[emailTemplate.emailType] | replace: '_':' ' | capitalize: true }}
+  </ng-container>
+  <ng-container appPageSubheaderSummary>
+    This is what the "{{ EmailTemplateTypeEnum[emailTemplate.emailType] | replace: '_':' ' | capitalize: true }}" email
```

Comment:
Feels like maybe the subheader should be `Preview of the {{ ... }} email notification` and then you probably don't need the second subheader.